app/src/main/kotlin/com/softteco/template/ui/feature/login/LoginScreen.kt

```diff
@@ -113,25 +112,23 @@ private fun ScreenContent(
                         if (state.loginState) {
                             onLoginClicked() // transfer to user's screen
                         }
-                    }
-                )
-                ClickableText(
-                    text = AnnotatedString(stringResource(id = R.string.sign_up)),
-                    onClick = {
-                        onSignUpClicked()
-                    },
-                    style = MaterialTheme.typography.bodyLarge
+                    })
+                SecondaryButton(
```

Comment:
Let's also add loading and preventing double-clicking here.

Reply:
By clicking this button we go to another screen. Without any requests. How to track loading status?

Reply:
You are right, for current purposes it doesn't need to, but it would be useful for the future, and it would be consistent to have the same features "loading, avoiding double clicking" for both primary and secondary buttons.